docs/source/ext/github_tools.py

```diff
@@ -40,6 +40,7 @@
 # Functions
 # ----------------------------------------------------------------------------
 
+req = urllib.request.Request('http://www.url.com')
```

Comment:
- `'http://www.url.com'` : Can you explain what is this url?
- Could you explain why `req` should be a global variable.

Reply:
yes, it has broken out the whole codebase. First of all, req must be a global variable to look at our URL through which we can fetch or use it. It will be better to know about the URL link which we are using it. Thank you

Reply:
As you can see [here](https://github.com/fury-gl/fury/blob/89aaa7eddc645979047c6740b24f86e489085aeb/docs/source/ext/github_tools.py#L45), `Request` does not to be global for doing this fix.

I still do not understand what is `http://www.url.com`.  Why this URL? why pointing to this place? when going to this url there is nothing.

Reply:
Url should be this one https://api.github.com/repos/{0}/stats/contributors. I think we can disclose or reject this PR for this issue. But in order to know the issue behind it, you can still use the [Bandit](https://github.com/PyCQA/bandit) tool to find out the vulnerabilities on the fury repository. Here is the [link.](https://github.com/PyCQA/bandit)